modules/ws/src/test/java/org/mule/module/ws/consumer/WSConsumerTestCase.java

```diff
@@ -66,6 +76,22 @@ public void failsToInitializeWithInvalidOperation() throws MuleException
         wsConsumer.initialise();
     }
 
+    @Test
+    public void outboundEndpointIsDisposed() throws Exception
+    {
+        TestEndpointFactory endpointFactory = new TestEndpointFactory();
+        muleContext.getRegistry().registerObject(OBJECT_MULE_ENDPOINT_FACTORY, endpointFactory);
+
+        WSConsumer wsConsumer = createConsumer();
+        wsConsumer.getConfig().setConnector(new HttpConnector(muleContext));
+        wsConsumer.initialise();
+
+        verify((Disposable) endpointFactory.getCreatedEndpoint(), never()).dispose();
```

Comment:
You're right, I only wanted to be sure that dispose were not called as a side effect of another operation.